frontend/src/components/pages/rp-connect/Pipelines.Create.tsx

```diff
@@ -84,10 +83,24 @@ class RpConnectPipelinesCreate extends PageComponent<{}> {
     const alreadyExists = pipelinesApi.pipelines.any((x) => x.id === this.fileName);
     const isNameEmpty = this.fileName.trim().length === 0;
 
+    const CreateButton = () => {
+      const toast = useToast();
+
+      return (
+        <Button
+          variant="solid"
+          isDisabled={alreadyExists || isNameEmpty || this.isCreating}
+          loadingText="Creating..."
+          isLoading={this.isCreating}
+          onClick={action(() => this.createPipeline(toast))}
```

Comment:
nit: I don't really like that toast is being passed as a parameter to create / update fns. Wouldn't it be cleaner to provide onSuccess / onFailure callbacks and trigger the toasts in here instead of `createPipeline`?